src/utils/__tests__/watch_stdio_for_line.js

```diff
@@ -0,0 +1,36 @@
+import execa from 'execa';
+import stripAnsi from 'strip-ansi';
+import sinon from 'sinon';
+
+import { watchStdioForLine } from '../watch_stdio_for_line';
+
+describe('src/utils/watch_stdio_for_line', function () {
+  const sandbox = sinon.sandbox.create();
```

Comment:
just a reminder - `sinon.sandbox.create()` is deprecated, please use `sinon.createSandbox()` instead, @tylersmalley :)